### PR TITLE
Player class

### PR DESCRIPTION
**Pull request by vlsulliv**

### CODE REVIEW

File: src/com/orderinchaos/Obstacle.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  public void setPass(boolean pass) {
+    this.pass = pass;
+  }
+
+  public int getAttempts() {
+    return attempts;
+  }
+
+  public void setAttempts(int attempts) {
```
Comment (wonil-park) [inline src/com/orderinchaos/Obstacle.java]: Probably needs to be private

File: src/com/orderinchaos/Riddle.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    this.question = question;
+    this.answer = answer;
+  }
+
+  // ON YOUR MARK
+  public String getQuestion() {
+    return question;
+  }
+
+  public void setQuestion(String question) {
```
Comment (wonil-park) [inline src/com/orderinchaos/Riddle.java]: Probably needs to be private, as well

File: src/com/orderinchaos/Riddle.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    this.question = question;
+  }
+
+  public String getAnswer() {
+    return answer;
+  }
+
+  public void setAnswer(String answer) {
```
Comment (wonil-park) [inline src/com/orderinchaos/Riddle.java]: Probably needs to be private here as well

File: src/com/orderinchaos/Game.java [hunk 4]
```diff
@@ -88,10 +89,13 @@ public Player changePhase(Room room, Player player) {
   }
 
   public void loadRooms() {
+
     Stream<String> descriptions = Util.TEXT_READER("room_descriptions.txt");
     descriptions.forEach(line -> {
       String[] tempArray = line.split("[|]");
-      roomList.add(new Room(tempArray[0], tempArray[1]));
+      // TODO: check if remove() returns value
```
Comment (wonil-park) [inline src/com/orderinchaos/Game.java:96]: Checked

File: src/com/orderinchaos/Obstacle.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+  private void setRiddle (Riddle riddle) {
+    this.riddle = riddle;
+  }
+
+  // BUSINESS METHODS
+  public void displayRiddle() {
+    int attemptsLeft = getAttempts();
+    Scanner sc = new Scanner(System.in);
+    System.out.println(riddle.getQuestion());
```
Comment (wonil-park) [inline src/com/orderinchaos/Obstacle.java:37]: Let's modify/overload our util scanner and handle user interaction for test purposes

File: src/com/orderinchaos/Riddle.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+  @Override
+  public String toString() {
+    return "Riddle{" +
+      "question='" + question + '\'' +
+      ", answer='" + answer + '\'' +
+      '}';
+  }
+
+  // BUSINESS METHODS
```
Comment (wonil-park) [inline src/com/orderinchaos/Riddle.java:48]: Recommend deleting this.

**Review by ZenMasterOwl — APPROVED** [review]
Looks good

**Review by wonil-park — APPROVED** [review]
Looks good to me!

**Pull request merged**